tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_pgm_dispatch.cpp

```diff
@@ -207,6 +217,29 @@ void initialize_program(tt_metal::Program& program, uint32_t run_cycles) {
         kg.start_coord = { kg.end_coord.x + 1, kg.end_coord.y };
         kg.end_coord = kg.start_coord;
     }
+
+    if (erisc_enabled_g) {
+        auto erisc_cores = device->get_active_ethernet_cores();
```

Comment:
Should it be `get_active_ethernet_cores(true)`?

Reply:
yes, I think so (just talked w/ David who debugged this bug in CI recently)

Reply:
will fix